tests/Unit/NumericalAlgorithms/LinearOperators/Test_PowerMonitors.cpp

```diff
@@ -119,10 +119,143 @@ void test_power_monitors_second_impl() {
   CHECK_ITERABLE_APPROX(test_power_monitors, expected_power_monitors);
 }
 
+void test_relative_truncation_error_impl() {
```

Comment:
can you compute the relative and absolute truncation error in the test function above, so you don't have to duplicate the setup? Same in the Python test.

Reply:
You were right, its exactly the same setting. I have merged the tests in one.

Reply:
Did this change maybe get lost in a rebase or something? I don't see it on GitHub :)

Reply:
Oh, I put the test for the absolute truncation error at the end of this test.

Reply:
Can you merge this into the `test_power_monitors_second_impl` function above to reduce the amount of duplicate setup code?